### PR TITLE
Fix new product link in quick access

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | New product link in quick access is not working when you are in twig template (wrong redirection) |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1598 |
| How to test? | Reinstall Prestashop and click the create new product in the quick access drop-down menu. |

### CODE REVIEW

**Comment by aleeks** [comment]
Hello @fatmaBouchekoua, I think this is not working, 2 reasons : 
- if you rename `admin-dev`, it's not working anymore
- you have to test all links on 2 system, `symfony controllers`, and `legacy`.

It's not working in all case. I'm on it, I make a lot of change for this problem.

Tell me if it's working in all case for you your solution. I'm on it.

**Comment by fatmaBouchekoua** [comment]
Hello @aleeks ,
I've tested and it's working in all cases (symfony and legacy).
You are right about renaming `admin-dev`, but even when you modify the path of the route `admin_product_new` it's not working anymore.

**Comment by aleeks** [comment]
@fatmaBouchekoua, 
I will test your fix, I've made a lot of tests for my debug :

- on symfony controllers,
    - symfony links - broken (index.php/product/form/product/form/new), you don't have it ?
    - normal links - ok
- on legacy controllers
    - symfony links - ok
    - normal links - ok

But, it's tricky.. I'm on regex feature for sf routes in order to find it after rewrite, because some problem are on route's registration, we have to rewrite url, remove token, etc.
If you want, we can see that on thursday, ok ?

File: install-dev/data/xml/quick_access.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
       <link>index.php?controller=AdminCartRules&amp;addcart_rule</link>
     </quick_access>
     <quick_access id="New_product" new_window="0">
-      <link>index.php/product/new</link>
+      <link>/admin-dev/index.php/product/new</link>
```
Comment (mickaelandrieu) [inline install-dev/data/xml/quick_access.xml]: are you sure ? :)

Comment (Shudrum) [inline install-dev/data/xml/quick_access.xml]: Yep, be careful the folder name is not fixed.

Comment (fatmaBouchekoua) [inline install-dev/data/xml/quick_access.xml]: I agree the folder name is not fixed, even the `/new` is not fixed also.
Ok I will think about what @aleeks said.
Thanks ..

**Pull request closed without merging**